### PR TITLE
Lgraber/mobile handling

### PR DESCRIPTION
**Pull request by LGraber**
Adds support for using refresh tokens on mobile devices so that we can maintain long lived connections and not ask the user to constantly log in.
Also adds a slight margin on mobile so that scrolling is easier

### CODE REVIEW

**Review by illonage — APPROVED** [review]

**Comment by illonage** [comment]
Hi @jcopenha, @ronzilla, and @johnnyyan - Can you review the PR tomorrow, please?

File: force-app/main/default/lwc/tableauViz/tableauViz.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+* Support scrolling on phones
+* With this style we create a slim margin on the right
+* that users can scroll the entire screen via. Without
+* this, if the viz takes up the entire screen, the user
+* will not be able to scroll to other parts of the SF
+* page
+*************************** */
+@media (max-width: 500px) {
+    .tabVizPlaceholder {
+        margin-right: 10px;
```
Comment (johnnyattableau) [inline force-app/main/default/lwc/tableauViz/tableauViz.css:11]: Just a question. How is this going to affect the viz? Will the viz adjust itself automatically?

**Review by johnnyattableau — APPROVED** [review]

File: force-app/main/default/lwc/tableauViz/__tests__/tableauViz.test.js [hunk 2]
```diff
@@ -325,4 +325,97 @@ describe('tableau-viz', () => {
...
+
+        it('Detects that this is not mobile', () => {
+            const appendFn = jest.fn();
+            const vizToLoad = {
+                searchParams: {
+                    append: appendFn
+                }
+            };
+
+            const userAgent = `Some random browser that doesn't use the SDK`;
```
Comment (Tableau-Ron) [inline force-app/main/default/lwc/tableauViz/__tests__/tableauViz.test.js:362]: Consider setting this to a legit User-Agent

**Review by Tableau-Ron — APPROVED** [review]

File: force-app/main/default/lwc/tableauViz/tableauViz.css (hunk 1 above)
Comment (LGraber) [reply force-app/main/default/lwc/tableauViz/tableauViz.css:11]: Yes. Just shrinks the div a little bit

File: force-app/main/default/lwc/tableauViz/__tests__/tableauViz.test.js (hunk 2 above)
Comment (LGraber) [reply force-app/main/default/lwc/tableauViz/__tests__/tableauViz.test.js:362]: will update in next push

**Pull request merged**